src/vs/workbench/api/node/extHostExtensionService.ts

```diff
@@ -40,6 +40,12 @@ class ExtensionMemento implements IExtensionMemento {
 			this._value = value;
 			return this;
 		});
+
+		this._storage.onDidChangeStorage(e => {
```

Comment:
leak? (if this would ever get disposed...)

Reply:
@jrieken via https://github.com/Microsoft/vscode/pull/61024/commits/134ca026674e22afb967636443811d252c3e1076#diff-c61682f360a18cf690ac59fe3f625db7